imap_processing/idex/idex_l1a.py

```diff
@@ -108,7 +108,7 @@ class PacketParser:
     .. code-block:: python
 
         from imap_processing.idex.idex_l1a import PacketParser
-        l0_file = "imap_processing/tests/idex/imap_idex_l0_sci_20230725_v001.pkts"
+        l0_file = "imap_processing/tests/idex/imap_idex_l0_sci_12142023_v001.pkts"
         l1a_data = PacketParser(l0_file, data_version)
         l1a_data.write_l1a_cdf()
     """
```

Comment:
Now that I am looking at this code from a different perspective, I am thinking this `Examples` section could be moved to the module docstring, instead of having it here in this class docstring. I think that might be a better location for it. When someone reads this code and wonders how to use it, having the example at the top of the file seems more intuitive. Your call!

Reply:
Agreed ! Makes sense to me.